### PR TITLE
[docs]: Fix documentation for Checkbox Component

### LINKED ISSUE #11053: [docs]: Update the documentation for the Checkbox Component
### The entire URL of the documentation with the issue

https://docs.tooljet.com/docs/widgets/checkbox

**Formatting updates:**
- Replace  "`fx`" with "**fx**" wherever applicable. ( i.e. remove the backticks (``) and bold -> **fx**)
- For each h2 (created using ##), remove 24px padding-bottom. The `div` that wraps up the h2 headers and the section below the header should only have a "padding-top" of 24px and no "padding-bottom". 

**Content updates:**
- Event Table
    - Update the content in the "Description" of the given 3 events - [reference](https://docs.tooljet.com/docs/widgets/text-input/#events) ( Please just check the sentence structure from the given reference, the rest should remain untouched )
    - Add a full stop at the end of the sentence in the "Description" column.
- Add a divider before the "Styles" section [reference](https://docs.tooljet.com/docs/widgets/textarea#styles)

**The changes are to be implemented in two files -** 
 Next - ToolJet/docs/docs/widgets/checkbox.md
 Version 2.50.0 (LTS) - ToolJet/docs/versioned_docs/version-2.50.0-LTS/widgets/checkbox.md

### If the issue is confirmed, would you be willing to submit a pull request?

No

### Code of Conduct

- [X] I agree to follow the ToolJet Code of Conduct

### PR DESCRIPTION
**Pull request by eyesee11**
will close #11053

### CODE REVIEW

**Comment by PriteshKiri** [comment]
Hey @eyesee11,

Thanks for your contribution.

Can you please update the PR name ( add a relevant PR name ) and link your issue to this PR.

File: docs/docs/widgets/checkbox.md [hunk 1]
```diff
@@ -1,129 +1,133 @@
 ---
-id: checkbox
-title: Checkbox
----
+
```
Comment (RudraPratik30) [inline docs/docs/widgets/checkbox.md:2]: Hey, you are not supposed to edit this section, can you please revert the changes from this section?

File: docs/docs/widgets/checkbox.md [hunk 2]
```diff
@@ -1,129 +1,133 @@
...
 
 ## Properties
 
 ### Data
 
 | Property       | Description    | Expected Value         |
 |:---------------|:---------------|:-----------------------|
-| Label          | The text to be used as the label for the checkbox.          | String (e.g., `Select payment preference`).                 |
-| Default status | Sets the default status when the app is loaded.                | Toggle the on/off switch or click on `fx` and dynamically set the value. |
+| Label          | The text to be used as the label for the checkbox.          | String (e.g., **Select payment preference**).                 |
```
Comment (RudraPratik30) [inline docs/docs/widgets/checkbox.md:18]: You have to remove backticks from fx only and rest of the backticks should be untouched, please make this change wherever necessary.

File: docs/docs/widgets/checkbox.md [hunk 3]
```diff
@@ -1,129 +1,133 @@
...
+<div style={{paddingTop:'24px'}}>
 
 ## Events
 
 | <div style={{ width:"100px"}}> Event </div>     | <div style={{ width:"100px"}}> Description </div> |
 |:----------- |:----------- |
-| On change | On change event is triggered when checkbox input is changed. |
-| On check (deprecated) | On check event is triggered when checkbox input is checked. |
-| On uncheck (deprecated)| On uncheck event is triggered when checkbox input is unchecked. |
+| On change | The on change event is triggered when the checkbox input is changed. |
```
Comment (RudraPratik30) [inline docs/docs/widgets/checkbox.md:29]: You need to update the content of this table referring to the reference given to you in the issue, you don't need to exactly copy from the reference but you have to frame the sentence in a similar way.

File: docs/docs/widgets/checkbox.md [hunk 4]
```diff
@@ -1,129 +1,133 @@
...
+| Border color    | Sets the color of the checkbox. | Select the color or click on **fx** and input code that programmatically returns a Hex color code.          |
+| Checked color    | Sets the color of the checkbox when it is checked. | Select the color or click on **fx** and input code that programmatically returns a Hex color code.          |
+| Unchecked color    | Sets the color of the checkbox when it is not checked. | Select the color or click on **fx** and input code that programmatically returns a Hex color code.          |
+| Handle color    | Sets the color of the checked symbol inside the checkbox. | Select the color or click on **fx** and input code that programmatically returns a Hex color code.          |
+| Box shadow      | Sets the box shadow properties of the component.                                              | Select the box shadow color and adjust related properties or set it programmatically using **fx**.                                            |
 
 </div>
 
-
+---
```
Comment (RudraPratik30) [inline docs/docs/widgets/checkbox.md:133]: Can you please remove this? we don't need a divider at the end of the doc

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]
Hey @eyesee11 please make the mentioned changes in both versions, thanks!

**Comment by eyesee11** [comment]
Hey Pratik!!
I'm really sorry for the inconvenience
Sure I'll adhere to those mentioned
I just need to ask if I have to make a new PR for both the files again or
just for this file.

**Comment by PriteshKiri** [comment]
Hey @eyesee11,

You can update the same PR.

**Comment by PriteshKiri** [comment]
Closing the PR due to lack of progress.

**Pull request closed without merging**